pkg/kubectl/cmd/create_deployment.go

```diff
@@ -153,7 +165,18 @@ func createDeployment(f cmdutil.Factory, cmdOut, cmdErr io.Writer,
 	generatorName = fallbackGeneratorNameIfNecessary(generatorName, resourcesList, cmdErr)
 
 	imageNames := cmdutil.GetFlagStringSlice(cmd, "image")
-	generator, ok := generatorFromName(generatorName, imageNames, deploymentName)
+	replicas := cmdutil.GetFlagInt(cmd, "replicas")
+
+	command := []string{}
+	argsArray := args[1:]
```

Comment:
I have experimented locally. If I were to change this to `args` without the `[1:]`, then the args to be executed would include the name of the deployment.

```bash
k create deploy my-deployment --image ubuntu '/bin/bash'
k get -o yaml deploy my-deployment
... 
    spec:
      containers:
      - args:
        - my-deployment
        - /bin/bash
        image: ubuntu
...
```

I'm trying to figure out a good way to write a test for this.